Assume Role panic.

Command
---

 make children

Error
---
panic: SharedConfigAssumeRoleError: failed to load assume role for arn:aws:iam::xxxxxxx:role/OrganizationAccountAccessRole, source profile has no shared credentials

State
-----

Success root provision 
Accounts: dev, prod, staging, testing

Users Created:
---

xxx-root-bootstrap

Roles Created
----

 xxx-root-admin
 xxx-root-bootstrap
xxx-root-readonly

Groups Created
----
xxx-dev-admin
xxx-prod-admin
xxx-staging-admin
xxx-testing-admin
xxx-root-admin
xxx-root-readonly

Other Info
------

/scripts/lib.sh: line 9: /artifacts/dev.env: No such file or directory is this normal ?

Debugging indicates that the aws config is set correctly as well as the profile. when running the command from within lib.sh

One moved the created .aws/config and .aws/credentials from the generated artifacts to my local env, I can run assume-role without error in this env.

Any Info would be cool, thanks

--- a/scripts/lib.sh
+++ b/scripts/lib.sh
@@ -32,12 +32,11 @@ function assume_role() {
 	# This is because the [`assume-role`](https://github.com/remind101/assume-role) cli does not respect the SDK environment variables.
 	export HOME="/artifacts"
 	export AWS_CONFIG_FILE="${HOME}/.aws/config"
+	export AWS_SHARED_CREDENTIALS_FILE="${HOME}/.aws/credentials"
 
 	# Unset AWS credential environment variables so they don't interfere with `assume-role`
 	unset AWS_ACCESS_KEY_ID
 	unset AWS_SECRET_ACCESS_KEY
-	## fix for https://github.com/cloudposse/reference-architectures/issues/8
-	export AWS_SHARED_CREDENTIALS_FILE="${HOME}/.aws/credentials"
 
 	# Fetch the Role ARN from the configuration
 	export TF_VAR_aws_assume_role_arn=$(crudini --get ${AWS_CONFIG_FILE} "profile ${AWS_DEFAULT_PROFILE}" role_arn)

--- a/scripts/lib.sh
+++ b/scripts/lib.sh
@@ -35,7 +35,9 @@ function assume_role() {
 
 	# Unset AWS credential environment variables so they don't interfere with `assume-role`
 	unset AWS_ACCESS_KEY_ID
-	unset AWS_SECRET_ACCES_KEY
+	unset AWS_SECRET_ACCESS_KEY
+	## fix for https://github.com/cloudposse/reference-architectures/issues/8
+	export AWS_SHARED_CREDENTIALS_FILE="${HOME}/.aws/credentials"
 
 	# Fetch the Role ARN from the configuration
 	export TF_VAR_aws_assume_role_arn=$(crudini --get ${AWS_CONFIG_FILE} "profile ${AWS_DEFAULT_PROFILE}" role_arn)

